tidy3d/material_library/material_library.py

```diff
@@ -144,17 +132,21 @@ class MaterialItem2D(MaterialItem):
 
 Ag_RakicLorentzDrude1998 = VariantItem(
```

Comment:
The series of `RakicLorentzDrude` model is based on the analytical model from the paper. There is probably no need to update them. What improvement have you observed with the new poles?

Reply:
I've not updated all of the `RakicLorentzDrude` materials as many of them really shows no relevant improvements. But the weighted RMS error of some ones was improved by at least 1 order or magnitude using the new fitter. 

For example:

Al-RakicLorentzDrude1998
![Al-RakicLorentzDrude1998](https://github.com/flexcompute/tidy3d/assets/125516370/44244020-ea97-4b73-8df5-42e9661abc3f)

Ag-RakicLorentzDrude1998
![Ag-RakicLorentzDrude1998](https://github.com/flexcompute/tidy3d/assets/125516370/0b978384-0761-42a2-9183-456380d62fc6)